data-access-analysis-tools/index-nb-intro.rst

```diff
@@ -0,0 +1,141 @@
+.. This is a template rst file (.rst) within the Vera C. Rubin Observatory Documentation for Data Preview 0.1 (DP0.1) documentation project. This template can be used for a directory's index.rst or other pages within the directory. This comment and proceeding blank line may be deleted after the file is copied and renamed within the destination directory.
+
+.. Review the README on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Do not delete the expected section title, reference or related comments provided from the template.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. This is the label that can be used for cross referencing this file.
+.. Recommended title label format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. Each section should include a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link and explicit title using the syntax :ref:`link text <label-name>`.
+.. A warning will alert you of identical labels during the linkcheck process.
+
+
+.. _Data-Access-Analysis-Tools-NB-Intro:
+
+#######################################
+Introduction to the RSP Notebook Aspect
+#######################################
+
+Most RSP users will find Jupyter Notebooks to be the most efficient and powerful way to interact with the DP0.1 data set.
+For DP0.1, Jupyter Notebooks will be the only way to access images. 
```

Comment:
This isn't strictly true as you have access to a shell and the stack inside JupyterLab which is still part of the notebook aspect.  Users could build a butler and run scripts from the shell if they wish, but maybe this is covered later.

Reply:
From what I understand we are trying not to overwhelm people with discussions of terminals and butler command-line tooling but as written it is true that accessing images is a notebook activity and not a VO tooling activity. On the command line you could run `butler retrieve-artifacts` to get the file so we might want to replace "only" with "primary" here.

Reply:
I agree just changing only --> primary would be fine.  The terminal is mentioned later